stable/ibex-dashboard/templates/storageclass.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClassName) -}}
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+ name: "{{ .Values.persistence.storageClass }}"
+provisioner: {{ .Values.persistence.storage.provisioner }}
+parameters:
+ skuName: {{ .Values.persistence.storage.parameters.skuName }}
+ location: {{ .Values.persistence.storage.parameters.location }}
+ {{- end -}}
```

Comment:
@itye-msft no newline at EOF here. Not sure why circleci lint didn't catch this one.

Reply:
Also this is not a large issue but there's an extra space on the line at the end, so it's still not recognized as a newline at EOF.

Reply:
Newline fixed 👍